webrtc.html

```diff
@@ -1297,6 +1299,25 @@
               </li>
 
               <li>
+                <p>If <var>candidate</var> is <code>null</code>, then run the
+                following steps:</p>
+                <ol>
+                  <li>
+                    <p>For each media section in the last successfully applied
```

Comment:
Done. The link to the RFC is in the "Terminology" section which this now links to.